src/test/ui/rfc-2091-track-caller/track-caller-ffi.rs

```diff
@@ -0,0 +1,50 @@
+// run-pass
+
+#![feature(track_caller)]
+
+use std::panic::Location;
+
+extern "Rust" {
+    #[track_caller]
+    fn tracked() -> &'static Location<'static>;
+    fn untracked() -> &'static Location<'static>;
```

Comment:
Can you use "namespaced" names just in case this could collide with something in the platform?

E.g. `rust_track_caller_ffi_test_untracked`.

Reply:
Fixed. :)